src/qweb/qweb.ts

```diff
@@ -676,24 +676,23 @@ export class QWeb extends EventBus {
 
     function handleProperties(key, val) {
       let isProp = false;
-      if (node.nodeName === "input" && key === "checked") {
-        let type = (<Element>node).getAttribute("type");
-        if (type === "checkbox" || type === "radio") {
+      if (node.nodeName === "input") {
+        if (key === "checked") {
+          let type = (<Element>node).getAttribute("type");
+          if (type === "checkbox" || type === "radio") {
+            isProp = true;
+          }
+        }
+        if (key === "value" || key === "readonly" || key === "indeterminate") {
```

Comment:
indeterminate not only for checkboxes and radio (for tagName input)? Also, it seems to be a valid attribute on `<progress>` tag as well.